en/docs/develop-components/sharing-and-reusing/use-a-connection-in-your-service.md

```diff
@@ -48,10 +48,20 @@ To integrate another service into your application, click the appropriate tab be
           | ChoreoAPIKey            | CHOREO_<CONNECTION_NAME\>CHOREOAPIKEY                          |
 
 
-          If you'd like to use custom environment variable names instead of the Choreo-defined ones, add the dependency as a service reference under `dependencies` in the same file. For more details, refer to the instructions under the `component.yaml file (v1.0)` tab.
+          If you'd like to use custom environment variable names instead of the Choreo-defined ones, add env configurations under `configuration` in the same file.
```

Comment:
```suggestion
          If you want to use custom environment variable names instead of the Choreo-defined ones, you must define the environment configurations under the `configuration` section in the same file.
```